mlflow/projects/utils.py

```diff
@@ -94,20 +91,21 @@ def _is_git_repo(path):
         return False
 
 
-def _is_local_uri(uri):
-    """Returns True if passed-in URI should be interpreted as a path on the local filesystem."""
-    if _GIT_URI_REGEX.match(uri):
-        return False
+def _parse_file_uri(uri: str) -> str:
+    """Converts file URIs to filesystem paths"""
+    parsed_uri = uri
+    if _is_file_uri(uri):
+        parsed_file_uri = urllib.parse.urlparse(str(uri))
```

Comment:
```suggestion
        parsed_file_uri = urllib.parse.urlparse(uri)
```

`str` here looks redudant.